spark/src/main/kotlin/com/adevinta/spark/tokens/PaletteTokens.kt

```diff
@@ -0,0 +1,121 @@
+ * SOFTWARE.
+ */
+
+package com.adevinta.spark.tokens
+
+import androidx.compose.ui.graphics.Color
+
+internal object PaletteTokens {
+    // Orange colors
+    internal val Orange10 = Color(0xFF351000)
+    internal val Orange20 = Color(0xFF571F00)
+    internal val Orange30 = Color(0xFF7B2F00)
+    internal val Orange40 = Color(0xFFA14000)
+    internal val Orange50 = Color(0xFFC95200)
+    internal val Orange60 = Color(0xFFFF6E14)
+    internal val Orange70 = Color(0xFFFF8C53)
+    internal val Orange80 = Color(0xFFFFB694)
+    internal val Orange90 = Color(0xFFFFDBCC)
+
+    // Blue colors
+    internal val Blue10 = Color(0xFF001C3B)
+    internal val Blue20 = Color(0xFF003060)
+    internal val Blue30 = Color(0xFF004787)
+    internal val Blue40 = Color(0xFF025EB0)
+    internal val Blue50 = Color(0xFF3378CB)
+    internal val Blue60 = Color(0xFF5292E7)
+    internal val Blue70 = Color(0xFF74ADFF)
+    internal val Blue80 = Color(0xFFA6C8FF)
+    internal val Blue90 = Color(0xFFD5E3FF)
+
+    // Misc. colors
+    internal val Red10 = Color(0xFF410001)
+    internal val Red20 = Color(0xFF690002)
+    internal val Red30 = Color(0xFF910909)
+    internal val Red60 = Color(0xFFDB4437)
+    internal val Red80 = Color(0xFFFA5A4B)
+    internal val Red90 = Color(0xFFFFDAD5)
+
+    internal val Green10 = Color(0xFF002205)
+    internal val Green20 = Color(0xFF00390C)
+    internal val Green30 = Color(0xFF005315)
+    internal val Green60 = Color(0xFF4E9850)
+    internal val Green80 = Color(0xFF8BD989)
+    internal val Green90 = Color(0xFFA6F5A2)
+
+// Black & White colors
+
+    internal val AlmostWhite = Color(0xFFFAFAFA)
+    internal val AlmostBlack = Color(0xC7000000)
+    internal val Black = Color(0xFF121212)
+    internal val BlackLight = Color(0xFF474747)
+
+    internal val Grey0 = Color(0xFF000000)
+    internal val Grey10 = Color(0xFF1A1C1E)
+    internal val Grey20 = Color(0xFF2F3033)
+    internal val Grey30 = Color(0xFF46474A)
+    internal val Grey40 = Color(0xFF5E5E62)
+    internal val Grey50 = Color(0xFF76777A)
+    internal val Grey60 = Color(0xFF909094)
+    internal val Grey70 = Color(0xFFABABAE)
+    internal val Grey80 = Color(0xFFC7C6CA)
+    internal val Grey90 = Color(0xFFE3E2E6)
+    internal val Grey95 = Color(0xFFF1F0F4)
+    internal val Grey99 = Color(0xFFFDFBFF)
+    internal val Grey100 = Color(0xFFFFFFFF)
+
+    internal val GreyBlue10 = Color(0xFF0d1d29)
+    internal val GreyBlue20 = Color(0xFF23323f)
+    internal val GreyBlue30 = Color(0xFF394856)
+    internal val GreyBlue40 = Color(0xFF51606e)
+    internal val GreyBlue50 = Color(0xFF697987)
+    internal val GreyBlue60 = Color(0xFF8393a2)
+    internal val GreyBlue70 = Color(0xFF9dadbd)
+    internal val GreyBlue80 = Color(0xFFb8c8d8)
+    internal val GreyBlue90 = Color(0xFFd4e4f5)
+    internal val GreyBlue95 = Color(0xFFe6f2ff)
+    internal val GreyBlue99 = Color(0xFFfcfcff)
+
+    // Brikke Colors
```

Comment:
What about Brikke? Should we mention it?

Reply:
Until we have the colors for Spark it's good imo